### PR TITLE
Feature#21 add styles to feature branches view

### LINKED ISSUE #21: Add styles to feature Branches View
}# REQUIREMENTS
In this issue all styles are going to be added in the Branches View

# DESIGN
Provide references to images of the design, how it looks like now, and how it should look like.

# Acceptance Criteria
List all required acceptance criteria
- [x] The final result should be similar to the design

### PR DESCRIPTION
**Pull request by mariormt17**
Add styles to feature branches view
 
## Description: Why did you change/implement that?

This PR contains styles for branches view

## Testing: Help me how can I test or look at the changes?

imagen
imagen
imagen

### CODE REVIEW

File: reloj-checador/app/views/companies/_form.html.erb [hunk 1]
```diff
@@ -1,27 +1,21 @@
...
+    <%end%>
   </div>
 
-  <div class="actions">
-    <%= form.submit %>
+  <div class="d-flex justify-content-center mt-3 ">
+    <%= form.submit :class => "btn btn-primary btn-lg" %>
   </div>
-<% end %>
+<% end %>
```
Comment (OswaldoPineda) [inline reloj-checador/app/views/companies/_form.html.erb]: add an extra final line here

File: reloj-checador/app/views/companies/edit.html.erb [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+      <%= render 'form', company: @company %>
+      <%= link_to companies_path, :class => "btn btn-info" do%>
+        <i class="fas fa-arrow-left"></i> Back
+      <% end %>
+      <%= link_to @company, :class => "btn btn-info" do%>
+        <i class="fas fa-eye"></i> Show
+      <% end %>
+    </div>
+  </div>
+</div>
```
Comment (OswaldoPineda) [inline reloj-checador/app/views/companies/edit.html.erb]: add an extra final line here

File: reloj-checador/app/views/companies/index.html.erb [hunk 3]
```diff
@@ -1,29 +1,39 @@
...
+                <%= link_to company, method: :delete, data: { confirm: 'Are you sure?' }, :class => "btn btn-danger" do %>
+                <i class="fas fa-trash"></i>Destroy
+              <% end %>
+            </td>
+          </tr>
+        <% end %>
+      </tbody>
+    </table>
+  </div>
+</div>
```
Comment (OswaldoPineda) [inline reloj-checador/app/views/companies/index.html.erb]: add an extra final line here

File: reloj-checador/app/views/companies/new.html.erb [hunk 4]
```diff
@@ -1,5 +1,11 @@
...
+  <div class="card p-5">
+    <h1 class="card-title text-center">New Company</h1>
+    <div class="card-body p-5">
+      <%= render 'form', employee: @company %>
+      <%= link_to companies_path, :class => "btn btn-info" do%>
+        <i class="fas fa-arrow-left"></i> Back
+      <% end %>
+    </div>
+  </div>
+</div>
```
Comment (OswaldoPineda) [inline reloj-checador/app/views/companies/new.html.erb]: add an extra final line here

File: reloj-checador/app/views/companies/show.html.erb [hunk 5]
```diff
@@ -1,23 +1,34 @@
...
-<%= link_to 'Back', companies_path %>
+  <div class="mt-5">
+    <%= link_to companies_path, :class => "btn btn-info" do%>
+      <i class="fas fa-arrow-left"> </i> Back
+    <% end %>
+    <%= link_to edit_company_path(@company), :class => "btn btn-info" do%>
+      <i class="fas fa-user-edit"></i> Edit
+    <% end %>
+  </div>
+</div>
```
Comment (OswaldoPineda) [inline reloj-checador/app/views/companies/show.html.erb]: add an extra final line here

**Review by OswaldoPineda — COMMENTED** [review]
You can install an extension to your IDE to add the extra line at the end automatically.

Also, check your commit titles to follow a better structure, you can find more information here: https://chris.beams.io/posts/git-commit/

let me know if there are any questions.

File: reloj-checador/db/schema.rb [hunk 6]
```diff
@@ -15,7 +15,6 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-
```
Comment (EduardoGHdez) [inline reloj-checador/db/schema.rb:18]: It doesn't look like we are creating a migration at all.... perhaps... did you miss to add it?

File: reloj-checador/app/views/companies/_form.html.erb (hunk 1 above)
Comment (mariormt17) [reply reloj-checador/app/views/companies/_form.html.erb]: Finished, thanks

File: reloj-checador/app/views/companies/edit.html.erb (hunk 2 above)
Comment (mariormt17) [reply reloj-checador/app/views/companies/edit.html.erb]: Finished, thanks

File: reloj-checador/app/views/companies/index.html.erb (hunk 3 above)
Comment (mariormt17) [reply reloj-checador/app/views/companies/index.html.erb]: Finished, thanks

File: reloj-checador/app/views/companies/new.html.erb (hunk 4 above)
Comment (mariormt17) [reply reloj-checador/app/views/companies/new.html.erb]: Finished, thanks

File: reloj-checador/app/views/companies/show.html.erb (hunk 5 above)
Comment (mariormt17) [reply reloj-checador/app/views/companies/show.html.erb]: Finished, thanks

File: reloj-checador/db/schema.rb (hunk 6 above)
Comment (mariormt17) [reply reloj-checador/db/schema.rb:18]: It's in another branch

Comment (EduardoGHdez) [reply reloj-checador/db/schema.rb:18]: Then, are these changes needed in this branch? 🤔

Comment (EduardoGHdez) [reply reloj-checador/db/schema.rb:18]: Can we just remove the commit? 😄

**Pull request merged**